### ISSUE TITLE
lowe instead of lower

### ISSUE THREAD

**Issue by bortzmeyer** [SIDN/CycleHunter#30]
In `findCyclicDep.py`, there is:

```
clearedZonesForOK.append(zone.lowe())
```

which is probably a typo.

**Comment by bortzmeyer** [comment]
```
Traceback (most recent call last):
  File "CycleHunter.py", line 54, in <module>
    find_cycles(output2, output3)
  File "/home/bortzmeyer/tmp/CycleHunter/findCyclicDep.py", line 1154, in find_cycles
    cyclic = sortDepsNew(zonesWoBailiwick)
  File "/home/bortzmeyer/tmp/CycleHunter/findCyclicDep.py", line 1024, in sortDepsNew
    clearedZonesForOK.append(zone.lowe())
AttributeError: 'str' object has no attribute 'lowe'
```

**Comment by gmmoura** [comment]
thanks for reporting, @bortzmeyer . 
this has been already fixed on #28
